.github/workflows/publish_release.yml

```diff
@@ -83,6 +83,11 @@ jobs:
         with:
           python-version: '3.11'
 
+      - name: Install build package
```

Comment:
(Probably a dopey question): Does this need [the GDAL wizardry](https://github.com/nasa/harmony-browse-image-generator/blob/main/.github/workflows/run_lib_tests.yml#L26-L30) that `run_lib_tests.yml` includes?

Reply:
We're going to find out.  I am not an expert at package building, but my few google searches said no and when I ran it with ACT it showed that it did build the package.  Of course I didn't test that, but the package should only have information about what it needs to build not the actual libraries no?

Reply:
Okay. Let me approve this and then we can give it a whirl!